### ISSUE TITLE
Accessibility:Step Input: The corresponding labels “value: 101” and “Min Max limits (min: -20, max: 20) are not read, when step input fields are accessed. 

### ISSUE THREAD

**Issue by puru-hk** [SAP/fundamental-ngx#3488]
Description: The corresponding labels “value: 101” and “Min Max limits (min: -20, max: 20) are not read, when step input fields are accessed. 

Expected: 
corresponding label should be read out when each & every UI is accessed. 

Screenshot:

![image](https://user-images.githubusercontent.com/32538291/94649552-84f6e600-0312-11eb-9d95-3979fbff8da4.png)

**Comment by puru-hk** [comment]
Tested it is working as expected.
